### PR TITLE
Implemented [SSH-7442]

### PR DESCRIPTION
**Pull request by kboratynski**
Determines whether it is possible to login with empty password using SSH.

### CODE REVIEW

File: include/tests_ssh [hunk 1]
```diff
@@ -96,6 +96,32 @@
...
+        logtext "Test: check PermitEmptyPasswords option"
+        FIND=`awk '/^PermitEmptyPasswords/ { print $2 }' ${SSH_DAEMON_CONFIG}`
+        FIND=`echo "${FIND}" | awk '{ print toupper($0) }'`
+        if [ "${FIND}" = "YES" ]; then
+            logtext "Result: PermitEmptyPasswords is enabled, login without password is possible"
+            Display --indent 4 --text "- SSH option: PermitEmptyPasswords" --result WARNING --color RED
+            ReportWarning ${TEST_NO} "M" "It is possible to login with empty password"
+            AddHP 0 3
+        elif [ "${FIND}" = "NO" ]; then
+            logtext "Result: PermitEmptyPasswords is disabled. Root can't login directly"
```
Comment (rhyven) [inline include/tests_ssh]: The PermitEmptyPasswords directive isn't related to root's ability to login.

Can I suggest changing this to:
"PermitEmptyPasswords is disabled. Users with blank passwords will be unable to login via SSH."

**Comment by kboratynski** [comment]
Sure. You are right. Modified commit. I just copied logtext. :-)

**Comment by kboratynski** [comment]
Moved into https://github.com/CISOfy/lynis/pull/100.

**Comment by kboratynski** [comment]
Moved into: https://github.com/CISOfy/lynis/pull/100.

**Pull request closed without merging**